src/urllib3/http2.py

```diff
@@ -163,19 +235,60 @@ def getresponse(  # type: ignore[override]
             data=bytes(data),
         )
 
+    def request(  # type: ignore[override]
+        self,
+        method: str,
+        url: str,
+        body: _TYPE_BODY | None = None,
+        headers: typing.Mapping[str, str] | None = None,
+        *,
+        preload_content: bool = True,
+        decode_content: bool = True,
+        enforce_content_length: bool = True,
+        **kwargs: typing.Any,
+    ) -> None:
+        """Send an HTTP/2 request"""
+        if "chunked" in kwargs:
+            # TODO this is often present from upstream.
+            # raise NotImplementedError("`chunked` isn't supported with HTTP/2")
```

Comment:
We should maintain this error, since chunked encodings aren't valid in HTTP/2.

Reply:
I Left this commented out, since while the tests pass, if you run the patch methods in this module and make a call to `urllib3.request`, it will always fail. Still need to follow this up the stream.

```
from urllib3.http2 import inject_into_urllib3
inject_into_urllib3()
import urllib3
urllib3.request('GET', url="https://www.google.com/")
```

```
NotImplementedError: `chunked` isn't supported with HTTP/2
```

Reply:
Sounds good, I created a new issue for tracking this: https://github.com/urllib3/urllib3/issues/3425